plugins/tech-insights-backend/src/service/router.ts

```diff
@@ -96,7 +97,29 @@ export async function createRouter<
         const entityTriplet = stringifyEntityRef({ namespace, kind, name });
         const checkResult = await factChecker.runChecks(entityTriplet, checks);
         return res.send(checkResult);
-      } catch (e) {
+      } catch (e: any) {
+        return res.status(500).json({ message: e.message }).send();
```

Comment:
I believe that if you just throw the error here it should be caught and formatted by our middleware? if you have ` router.use(errorHandler());`

Reply:
Yeah, that simplify things!